File: java/problem_1/problem1.java
public class problem1 
{
	public static void main(String[] args)
	{
		Integer i;
		Integer num = 0;
			
		for(i = 1; i < 1000; i++)
		{
			if(i%3 == 0 || i % 5 == 0)
				num += i;			
		}
		System.out.println("Answer: " + num);
	}	
}

File: c/problem_1/problem1.c
#include <stdio.h>

int main()
{
    int i;
    int num = 0;

    for(i = 1; i < 1000; i++)
    {
        if(i % 3 == 0 || i % 5 == 0)
            num += i;
    }

    printf("Answer: %d\n",num);
    return 0;
}
